### PR TITLE
build changeo container extended base

### PR DESCRIPTION
**Pull request by ggabernet**
Sorry, it's me again. I would need the extended image as I have internet access problems on the singularity container otherwise :)

### CODE REVIEW

**Comment by bgruening** [comment]
But this is pointing to a configuration issue on the Singularity side isn't it? Is an arbritrary  extended-base singularity container working?

**Comment by ggabernet** [comment]
It seems to be related to this issue: https://github.com/bioconda/bioconda-recipes/issues/11583

**Comment by ggabernet** [comment]
Tested it with a bioconda recipe that was previously using the non-extended base, and then updated to the extended base:
```
wget https://depot.galaxyproject.org/singularity/busco:1.2--py35_0
--2021-05-12 09:17:45--  https://depot.galaxyproject.org/singularity/busco:1.2--py35_0
Resolving depot.galaxyproject.org (depot.galaxyproject.org)... 128.118.250.14
Connecting to depot.galaxyproject.org (depot.galaxyproject.org)|128.118.250.14|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 266571776 (254M) [application/octet-stream]
Saving to: ‘busco:1.2--py35_0’

100%[===============================================================================================================================>] 266,571,776 5.50MB/s   in 43s

2021-05-12 09:18:29 (5.89 MB/s) - ‘busco:1.2--py35_0’ saved [266571776/266571776]

(base) [qeaga01@ilogin02 container]$ singularity shell busco\:1.2--py35_0
WARNING: Skipping mount /var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
Singularity> wget ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/database -P database
wget: bad address 'ftp.ncbi.nih.gov'
```

The latest version has an extended base and there is not this problem eny more of mounting `/etc/resolv.conf` in the container:
```
singularity shell busco\:5.1.2--py_0
Singularity> ping google.com
bash: ping: command not found
wget ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/database -P database
--2021-05-12 09:41:45--  ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/database
           => ‘database/.listing’
Resolving ftp.ncbi.nih.gov (ftp.ncbi.nih.gov)... 165.112.9.228, 130.14.250.12, 2607:f220:41e:250::10, ...
Connecting to ftp.ncbi.nih.gov (ftp.ncbi.nih.gov)|165.112.9.228|:21... connected.
Logging in as anonymous ... Logged in!
```
And it is able to do the download

**Comment by ggabernet** [comment]
Hi @bgruening, would it be possible that I test if this extended-base image would solve the internet connection problems?

File: combinations/hash.tsv [hunk 1]
```diff
@@ -130,4 +130,5 @@ star=2.6.1d,samtools=1.10
 star=2.6.1d,samtools=1.10,gawk=5.1.0
 planemo=0.74.3,git=2.30.2,galaxyxml=0.4.14
 minimap2=2.18,samtools=1.12
-changeo=1.0.2,igblast=1.15.0,wget=1.20.1
\ No newline at end of file
+changeo=1.0.2,igblast=1.15.0,wget=1.20.1
+changeo=1.0.2,igblast=1.15.0,wget=1.20.1	bioconda/extended-base-image:latest	0
```
Comment (mvdbeek) [inline combinations/hash.tsv:134]:
```suggestion
changeo=1.0.2,igblast=1.15.0,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	1
```

Comment (mvdbeek) [reply combinations/hash.tsv:134]: This should be the current base image bioconda is using. If an entry doesn't specify a build, that build is 0, so we need to set the build column to 1 here.

**Comment by mvdbeek** [comment]
Thanks for the link @ggabernet. Can you check if https://quay.io/repository/biocontainers/mulled-v2-7d8e418eb73acc6a80daea8e111c94cf19a4ecfd?tag=e5bd866a6803c301bf10b82e697c5dd2e49810a1-1&tab=tags works for you ?

**Comment by ggabernet** [comment]
@mvdbeek This base image worked, thanks for your recommendation!

**Pull request merged**